packages/@okta/vuepress-site/docs/guides/build-sso-integration/create-your-app/saml2/create.md

```diff
@@ -1,73 +1,34 @@
-          1. Select the **Contact app vendor** check box if Okta needs to contact you to enable SAML for the app. If you select this check box, you need to provide further general information about your app to our app analyst team.
-
-* In the new application window, you can specify **General Settings** and **Sign On** options, as well as assign the app to users in your org. Click **Edit** if you need to change any of the options, and **Save** when you have made your changes.
-
-You are now ready to configure and test your SAML app. You will need to obtain the Identity Provider metadata from within the app that you just created.
-
-1. Inside your application's settings on the Okta Admin Console, select the **Sign On** tab.
-1. In the **SIGN ON METHODS** section, locate the **Identity Provider metadata** link in the note above the **CREDENTIALS DETAILS** section.
-1. Right-click the **Identity Provider metadata** link and select **Copy Link Address**. This metadata link contains the information that you need to configure SAML in your SAML SP application.
-1. Follow the SAML SP app instructions on how to configure the Identity Provider metadata. We recommend using the Identity Provider metadata link to dynamically configure the metadata. If your SP doesn't support dynamic configuration, click **View Setup Instructions**. A new browser tab launches with the information that you need:
-    * Identity Provider Single Sign-On URL
-    * Identity Provider Issuer
-    * X.509 Certificate
-
-Copy this information and use it to configure the Identity Provider metadata in your SAML SP app.
-
-## Use SAML tool-kits
-
-If you have an existing SAML app where you want to add SSO, the following open source tool kits are another way to help you implement the SAML 2.0 specification for the WebSSO Profile for Service Providers using different programming languages:
-
-* [.NET Framework](https://en.wikipedia.org/wiki/.NET_Framework_version_history) 4.5 or later: [Kentor Authentication Services](https://github.com/KentorIT/authservices#kentor-authentication-services)
-* .NET Framework 4.0 or earlier: [ComponentSpace SAML 2.0 for ASP.NET and ASP.NET Core](https://www.componentspace.com/) - Paid software, single developer licenses start at $99
-* Java: [OpenSAML](https://wiki.shibboleth.net/confluence/display/OS30/Home), which is part of the [Shibboleth Development Project](https://www.shibboleth.net/)
-* Java: [Spring Security SAML](/code/java/spring_security_saml)
-* PHP: [SimpleSAMLphp](/code/php/simplesamlphp)
-* Python: [PySAML2](/code/python/pysaml2)
-* Ruby: [Ruby-SAML](https://rubygems.org/gems/ruby-saml)
+          1. For the check boxes that appear, select the **App type** check box if your company created the integration and it will never be released publicly. If you check this box, you don't need to enter any further information.
```

Comment:
avoid future tense: ...created the integration and it won't be released publicly.